libs/ledger-live-common/src/exchange/swap/getExchangeRates.ts

```diff
@@ -40,19 +35,21 @@ const getExchangeRates: GetExchangeRates = async (
   const amountFrom = transaction.amount;
   const tenPowMagnitude = new BigNumber(10).pow(unitFrom.magnitude);
   const apiAmount = new BigNumber(amountFrom).div(tenPowMagnitude);
+  const providers = await getProviders();
```

Comment:
This API call is the mostly costly one, it is already done when loading the swap pane. Isn't there a way to reuse to data ?

(in reality, results are cached using private cache but I think it would be better to explictly reuse it)

Reply:
done, we are reusing it now.